0.25.0/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "ramda.github.io",
+  "version": "0.25.0",
+  "description": "Ramda website",
+  "engines": {
+    "node": ">=6.0.0"
+  },
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ramda/ramda.github.io.git"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "bootstrap": "3.3.x",
+    "gitbook-cli": "2.3.x",
+    "handlebars": "4.x.x",
+    "highlight.js": "9.x.x",
+    "http-server": "0.9.x",
+    "jsdoc": "3.4.x",
+    "less": "2.7.x",
+    "less-plugin-autoprefix": "1.5.x",
+    "less-plugin-clean-css": "1.5.x",
+    "marked": "0.3.x",
+    "pug": "^2.0.0-beta6",
+    "ramda": "0.25.0",
```

Comment:
should we make this `latest`?

Reply:
I think it is nice to have the exact version here..